lib/miniproxy.rb

```diff
@@ -4,6 +4,7 @@
 
 module MiniProxy
   DEFAULT_DRB_SERVICE_TIMEOUT = 5
+  GET_TIME = Process.method(:clock_gettime)
```

Comment:
Approach copied from the Timeout library https://github.com/ruby/timeout/blob/master/lib/timeout.rb#L139-L142